### DISCUSSION TITLE
Relation of colour with grain orientation and File Structure/ IPF Triangle/ Pole Figure

### DISCUSSION THREAD

**Discussion by dikshanshs** [neperfepx/neper#244] [Q&A] [answered]
I am working on Neper and Damask for faster and efficient results. **_Help me understanding file structure_** (like .tess have seed position, faces and vertex position). And if there is a **_relation between colour and orientation_** let me know.

Edit: How to print IPF triangle and Pole figure using Neper?

**Comment by smallrain1996** [comment]
For the first question, you can learn the neper help documentation yourself to get this information. For the second question, you can refer to "- datacellcolscheme" in 5.1.4.

**Comment by CsatiZoltan** [comment]
@dikshanshs You can plot the crystal orientations as described in https://github.com/rquey/neper/issues/249#issuecomment-864432928
Concerning the IPF triangle and pole figures, I don't think they are possible in Neper. Use other tools or write your own one.

**Comment by dikshanshs** [comment, marked as answer]
Answer to Question 1 and 2
> For the first question, you can learn the neper help documentation yourself to get this information. For the second question, you can refer to "- datacellcolscheme" in 5.1.4.

> 

> 

Answer to IPF Triangle and Pole figure
> @dikshanshs You can plot the crystal orientations as described in #249 (comment)
Concerning the IPF triangle and pole figures, I don't think they are possible in Neper. Use other tools or write your own one.

Give output orientation using -for ori and use other tools like ATEX for plotting IPF Triangle and pole figure
